lib/devicetrust/native/device_darwin.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package native
+
+// #cgo CFLAGS: -Wall -xobjective-c -fblocks -fobjc-arc -mmacosx-version-min=10.13
```

Comment:
`-mmacosx-version-min=10.13` is the same as https://github.com/gravitational/teleport/blob/4e1c39639edf1ab494dd7562844c8b277b5cfa18/lib/auth/touchid/api_darwin.go#L20 - it's what the APIs we use call for (afaict).